admin-dev/themes/default/js/bundle/default.js

```diff
@@ -2,8 +2,20 @@
  * Default layout instanciation
  */
 $(document).ready(function() {
-    $('[data-toggle="tooltip"]').tooltip();
-    rightSidebar.init();
+  var $this = $(this);
+  var $ajax_spinner = $('#ajax_spinner');
```

Comment:
Please remember to use camelCase instead of snake_case (`$ajax_spinner` --> `$ajaxSpinner`)